src/VisualStudio/Core/Def/Implementation/DebuggerIntelliSense/DebuggerTextView.cs

```diff
@@ -364,6 +364,22 @@ public void DisconnectFromIntellisenseControllers()
             }
         }
 
+        public void QueuePostLayoutAction(Action action)
+        {
+            throw new NotImplementedException();
+        }
+
+        public bool TryGetTextViewLines(out ITextViewLineCollection textViewLines)
+        {
+            throw new NotImplementedException();
+        }
+
+        public bool TryGetTextViewLineContainingBufferPosition(SnapshotPoint bufferPosition, out ITextViewLine textViewLine)
```

Comment:
GetTextViewlineContainingBufferPosition can return null, and this method should return false in that case. it would be better to do this:
{
bool success= _inner.TryGetTextViewLineContainingBufferPosition(bufferPosition, out textViewLine);
return success;
}

Reply:
added a TODO waiting for TextViewExtensions update

---
In reply to: [223820128](https://github.com/dotnet/roslyn/pull/29016#discussion_r223820128) [](ancestors = 223820128)

Reply:
Fix now in: https://devdiv.visualstudio.com/DevDiv/Editor/_git/VS-Platform/commit/25b7188c54f0cdf6a5be87eeb38e3c6046d5788e?refName=refs%2Fheads%2Fv16.0-preview.1